### ISSUE TITLE
Don't delete original mime icons

### ISSUE THREAD

**Issue by FlorianMoser** [aimeos/aimeos#21]
When adding a product image and the deleting it, the image is (as expected) also deleted.

However when adding a non-image file, a mime icon is displayed. Deleting that media item also deletes the original Aimeos mime icon.

How to reproduce:

1. Open a product
2. add a media file (in my case a .zip file)
3. save -> Aimeos now displays the icon public/packages/aimeos/shop/mimeicons/application/zip.png
4. remove the media file
5. save

After step 5, the file public/packages/aimeos/shop/mimeicons/application/zip.png is gone.

**Comment by aimeos** [comment]
We will release the new 2019.10 version based on Laravel 6 that shouldn't have that problem any more next Thusday or Wednesday

**Comment by FlorianMoser** [comment]
Cool, thanks.
I currently also have the problem that I can't reset a password. A remember_token is generated and sent by email, but not stored in the DB. Is this also related with Laravel 6 or another issue?

**Comment by aimeos** [comment]
The column `remember_token` is for something different ("remember me" without login feature) then a password reset (new password is set in `password` column) in Laravel AFAIK

**Comment by FlorianMoser** [comment]
Yeah, you are right. I checked the wrong table. The reset token is stored in password_resets table. 
I had an issue with my routes. Nothing to do with Aimeos, sorry.
